base/dcp_feed.go

```diff
@@ -279,10 +283,7 @@ func (r *DCPReceiver) GetMetaData(vbucketId uint16) (
 // it's called, logs warning and does a hard reset on metadata for the vbucket
 func (r *DCPReceiver) Rollback(vbucketId uint16, rollbackSeq uint64) error {
 	Warnf(KeyAll, "DCP Rollback request.  Expected RollbackEx call - resetting vbucket %d to 0.", vbucketId)
-
-	// TODO: move to per-db stats, or if that's too difficult due to package dependencies, then at least under Global stats
-	dcpExpvars.Add("rollback_count", 1)
-
+	r.dbStatsExpvars.Add("dcp_rollback_count", 1)
```

Comment:
It looks like there are still some usages of the previous dcpExpvars remaining - any reason those shouldn't be migrated along with these to the new location?

Reply:
Not really. Just that was the only one commented. I guess the others fit fine in the per-db too?

Reply:
@JRascagneres yeah all can go under the per-db stats